backend/routes/drop.js

```diff
@@ -0,0 +1,49 @@
+const env = require('../env')
+const jwt = require('jsonwebtoken')
+const auth = require('../middleware/auth')
+const express = require('express')
+const router = express.Router()
+const courseData = require('../data/courses')
+const userData = require('../data/users')
+const mongoCollections = require('../config/mongoCollections');
+const users = mongoCollections.users;
+let { ObjectId } = require('mongodb');
+
+async function getRole(userId)  {
+    const usersCollection = await users();
+    let objId = ObjectId(userId);
+    const user = await usersCollection.findOne({ _id: objId });
+
+    if(!user) throw "User doesn't exist!";
+    
+    return user.role;
+}
+
+router.post('/', auth, async (req, res) => {
+    const { userId, courseId } = req.body
+
+    let role = await getRole(userId);
+    let courseName = "";
+
+    try {
+        if(role.toLowerCase() == 'student')   {
+            courseName = await courseData.removeStudent(courseId, userId);
+        }
+        else {//user is a teacher
+            courseName = await courseData.removeTeacher(courseId, userId);
+        }
+        
+        let userName = await userData.drop(courseId, userId);
+
+        res.sendStatus(200);
+        console.log(userName + ' has dropped ' + courseName + '!');
+        return;
```

Comment:
The front end won't be able to see `console.log` rather we can send a JSON response to confirm the action has occurred. This will be really helpful when we start working in the front end and want to make changes based on the user input. 
```suggestion
        res.status(200).json({dropped: true});
        console.log(userName + ' has dropped ' + courseName + '!');
```